src/main/java/prefs/GlobalConfig.java

```diff
@@ -82,6 +84,14 @@ public void addBoard(String name, List<PanelInfo> panels) {
     public void removeBoard(String name) {
         savedBoards.remove(name);
     }
+    
+    public void setLastOpenBoard(String board) {
```

Comment:
Similarly, there needs to be a way for callers to indicate that there is no last-opened board. You could either have it take an `Optional` or provide `clearLastOpenBoard`.